fetch-jars

```diff
@@ -3,26 +3,62 @@
 set -eu
 
 # Usage: ./fetch-jars [rev-name] [--copper]
+# If a revision is not specified, the script fetches the latest jars from the current branch,
+# falling back to develop.
+# If --copper is specified, only fetch the Copper jars.
 
+COMMIT_ARTIFACTS="https://foundry.remexre.xyz/commit-artifacts/"
 
+function has_jars {
+  wget --spider -q "$COMMIT_ARTIFACTS/$1"
+}
+
+rev=""
+
+# Look for jars in the specified revision
 if [[ $* != *--copper* && $# -gt 0 || $# -gt 1 ]]; then
-    rev=$(git rev-parse $1)
-    echo "Warning: Fetching unstable jars!  (commit $rev)"
+  rev=$(git rev-parse $1)
```

Comment:
Is `./fetch-jars --copper v0.4.4` intended to be supported? might make more sense to do to parse the args and provide usage up-front, then do the rest of the script...... or rewrite it in python and use argparse lol

Reply:
Yeah that doesn't work at the moment. I don't really feel like rewriting it in Python, and bash getopts is ugly/feels like overkill. I might try to fix that issue though.